ci_framework/hooks/playbooks/validate_podified_deployment.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Make sure Nova api service is up and running
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        oc get pods --selector service=nova-api -o jsonpath={.items[*].status.phase}
```

Comment:
I think it would be better if we specify the namespace in the oc command itself.